documents/contributing.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page
+title: Help
+subtitle: Providing Feedback and Contributing New Material
+---
+
+### Providing feedback and getting help
+
+- [Provide feedback or get help](https://github.com/datacarpentry/semester-biology/issues/new) by opening a new issue at the 'Data Carpentry for Biologists' repository on GitHub (*You'll have to be logged into GitHub*). 
+
+- Please, provide a clear description of your question, comment, or proposed change in the `Title` section and use the `Leave a comment` section for further detail or discussion. Select `Submit new issue` and the repository maintainers will be notified of your feedback. Thanks!
+
+- If it is necessary for you to send your note to us some other way, please mail 
+us at _LINK TO EMAIL_. We want hear from you and we want it to be easy for you.
```

Comment:
How about just making this "You can also email us at". You can use datacarpentrysemester@weecology.org for the email address.